collections/_version2-english/9-fork/2-to-dos.md

```diff
@@ -0,0 +1,70 @@
+---
+title: To-do's
+---
+{% capture glossary_path %}{{ "/" | absolute_url }}/{{page.collection}}/7-glossary.html{% endcapture %}
+{% assign glossary_child_universe = glossary_path | append: '#Child_Universe' %}
+{% assign glossary_final_outcome = glossary_path | append: '#Final_Outcome' %}
+{% assign glossary_finalized_market = glossary_path | append: '#Finalized_Market' %}
+
+{% assign glossary_fork = glossary_path | append: '#Fork' %}
+{% assign glossary_forking_market = glossary_path | append: '#Forking_Market' %}
+{% assign glossary_forking_period = glossary_path | append: '#Forking_Period' %}
+{% assign glossary_locked_universe = glossary_path | append: '#Locked_Universe' %}
+{% assign glossary_losing_universe = glossary_path | append: '#Losing_Universe' %}
+{% assign glossary_parent_universe = glossary_path | append: '#Parent_Universe' %}
+{% assign glossary_reputation_token = glossary_path | append: '#Reputation_Token' %}
+{% assign glossary_repv1 = glossary_path | append: '#REPv1' %}
+{% assign glossary_repv2 = glossary_path | append: '#REPv2' %}
+{% assign glossary_roi = glossary_path | append: '#ROI' %}
+{% assign glossary_sibling_universe = glossary_path | append: '#Sibling_Universe' %}
+{% assign glossary_winning_universe = glossary_path | append: '#Winning_Universe' %}
+
+# To-do's
+If you are a REP holder, there are some things you **MUST** do for a [fork]({{glossary_fork}}) to not lose your [REP]({{glossary_reputation_token}}).
+
+## Before a fork starts
+REP holders have 2 things to do even before a [fork]({{glossary_fork}}) starts so that they don't lose the value of their [REP]({{glossary_reputation_token}}). One is to [convert REPv1 to REPv2](#convert-repv1-to-repv2) and the other is to [check monthly to see if a fork occurs](#check-monthly-to-see-if-a-fork-occurs).
+
+### Convert REPv1 to REPv2
+REPv1 holders **MUST** convert their [REPv1]({{glossary_repv1}}) to [REPv2]({{glossary_repv2}}). Because REP in the [parent universe]({{glossary_parent_universe}}) can be migrated to only its [child universe]({{glossary_child_universe}}).  (The name of REP version 1 is `REP` correctly, but to distinguish from version 2, we call it `REPv1` here.)
```

Comment:
I recommend moving the comment about REPv1 being REP into the glossary definition of REPv1 rather than including it inline here.

Reply:
>I recommend moving the comment about REPv1 being REP into the glossary definition of REPv1 rather than including it inline here.

I will add the definition of `REPv1` to the glossary (https://github.com/msagansk/Augur.Guide/issues/37).
And the removal of that comment will be accomplished by committing [this suggestion](https://github.com/msagansk/Augur.Guide/pull/34#discussion_r557016088).

Reply:
The remaining task was saved as the issue https://github.com/msagansk/Augur.Guide/issues/37, so I'm closing this conversation.